walkthroughs/3a-low-code-api-development/walkthrough.adoc

```diff
@@ -317,7 +317,7 @@ The *Integration Flow Editor* should now list your Slack connection with a *Data
 
 . From the *Data Mapper* screen click the *body* element in the *Source* panel to expand it.
 
-. Click the *name* field under the *body*, then click the *message* element in the *Target* panel. This maps the value of the incoming HTTP request *body* to the outgoing Slack *message* property.
```

Comment:
@evanshortiss  I'm not sure how to update the second sentence after this change

Reply:
Perhaps something like the below?

```
. Click the *name* field under the *body*, then click the *body* element in the *Target* panel. This maps the value within the incoming HTTP request to the outgoing HTTP response.

```

Reply:
@finp, so `body` in the `Target` panel isn't a top-level property with a `message` inside? If not I think it would be fine to say. `This maps the *name* value from the incoming HTTP request *body* to the *body* of the outgoing Slack message.

Reply:
@damienomurchu that is so generic, does it mean anything?
@evanshortiss  that's good, implemented.